src/test/java/io/datanerds/avropatch/PatchTest.java

```diff
@@ -22,7 +22,7 @@
 public class PatchTest {
 
     private final UUID uuid = UUID.randomUUID();
-    private final Map<String, ?> headers = ImmutableMap.of("test 1", "foo", "test 2", uuid);
+    private final Map<String, ? super Object> headers = ImmutableMap.of("test 1", "foo", "test 2", uuid);
```

Comment:
Indeed, it's also in the API. I somewhat tried to restrict the value types to the ones supported by our Avro schema, but since that is not possible you found the leftovers from that experiment. Thanks!